app/javascript/styles/buttons.scss

```diff
@@ -0,0 +1,109 @@
+// Button styling.
+
+@import './bootstrap';
+@import './_variables';
+
+// btn-primary
+.btn-primary {
+  @extend .btn-primary;
+  background-color: $daria-color;
+  border: none;
+  color: $white;
+  margin-bottom: 2px;
+  font-size: $font-size;
+}
+
+.btn-primary:hover {
+  background-color: lighten($daria-color,15%);
+}
+
+.btn-primary:focus {
+  background-color: lighten($daria-color,15%);
+  color: $white;
+  text-decoration: none;
+}
+
+// btn-danger
+.btn-danger {
+  background-color: $dark-red !important;
+  border-color: $dark-red !important;
+}
+
+// btn-warning
+.btn-warning {
+  color: $black;
+  border-color: $daria-color;
+  background-color: $white;
+
+  &:hover {
+    background-color: $daria-color-lt;
+    border-color: $daria-color;
+  }
+  &:active {
+    background-color: $soft-blue !important;
+  }
+  &:focus {
+    background-color: $daria-color;
+    &:hover {
+      background-color: $daria-color-lt;
+    }
+  }
+}
+
+// btn-success
+.btn-success {
+  background-color: $daria-color;
+  border-color: $daria-color;
+
+  &:hover {
+    background-color: $daria-color-lt;
+    border-color: $daria-color;
+  }
+  &:active {
+    background-color: $soft-blue !important;
+  }
+  &:focus {
+    background-color: $daria-color;
+    &:hover {
+      background-color: $daria-color-lt;
+    }
+  }
+}
+
+// btn sizes
+.btn-lg {
```

Comment:
Before: 
![image](https://user-images.githubusercontent.com/3866868/122292925-7febbe00-cec4-11eb-8b0a-864c90e811e9.png)

After: TK

Reply:
After: ![image](https://user-images.githubusercontent.com/3866868/122293910-82024c80-cec5-11eb-89c8-1b144d8c203a.png)

(the spacing on this is a few pixels off but not substantially different)